models/intermediate/daily_metrics/int_pendo__feature_daily_metrics.sql

```diff
@@ -0,0 +1,71 @@
+with feature_event as (
+
+    select 
+        *,
+        cast( {{ dbt_utils.date_trunc('day', 'occurred_at') }} as date) as occurred_on
+
+    from {{ ref('pendo__feature_event') }} -- or should i use the source table?
```

Comment:
The grain of the `pendo__feature_event` should be the same as the source. With that I say we use the reference you have as this will also keep the DAG nice and clean.